yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -770,31 +771,45 @@ func getAssessmentReportContentFromAnalyzeSchema() error {
 	return nil
 }
 
-func filterIssuesForUnsupportedFeature(featureName string, issueReason string, schemaAnalysisReport utils.SchemaReport, unsupportedFeatures *[]UnsupportedFeature) {
+func filterIssuesWithObjectNamesForUnsupportedFeature(featureName string, issueReason string, schemaAnalysisReport utils.SchemaReport, unsupportedFeatures *[]UnsupportedFeature) {
```

Comment:
The function name is a bit confusing. How about something like `addUnsupportedFeaturesFromSchemaAnalysisReport` ?